aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIO.java

```diff
@@ -101,60 +100,59 @@ public S3FileIO() {
   }
 
   /**
-   * Constructor with custom s3 supplier and default AWS properties.
+   * Constructor with custom s3 supplier and S3 fileIO properties.
    *
    * <p>Calling {@link S3FileIO#initialize(Map)} will overwrite information set in this constructor.
    *
    * @param s3 s3 supplier
    */
   public S3FileIO(SerializableSupplier<S3Client> s3) {
-    this(s3, new AwsProperties());
+    this(s3, new S3FileIOProperties());
   }
 
   /**
-   * Constructor with custom s3 supplier and AWS properties.
+   * Constructor with custom s3 supplier and S3 fileIO properties.
    *
    * <p>Calling {@link S3FileIO#initialize(Map)} will overwrite information set in this constructor.
    *
    * @param s3 s3 supplier
-   * @param awsProperties aws properties
+   * @param s3FileIOProperties S3FileIOProperties
```

Comment:
nit: should use descriptive language like "S3 FileIO properties" instead of directly using VariableNames in javadoc.